ga4gh/cli.py

```diff
@@ -266,10 +269,10 @@ def server_main(parser=None):
             description="GA4GH reference server")
     addGlobalOptions(parser)
     args = parser.parse_args()
-    frontend.configure(args.config_file, args.config)
+    frontend.configure(args.config_file, args.config, args.oidc_config_file)
     frontend.app.run(
-        host="0.0.0.0", port=args.port,
-        use_reloader=not args.dont_use_reloader)
+        host="localhost", port=args.port,
+        use_reloader=not args.dont_use_reloader, ssl_context="adhoc")
```

Comment:
Actually, if it turns out we can use the central Flask config file for the OIDC stuff, I think we should just make a boolean flag CLI argument to turn on and off SSL, which defaults to off.